bin/install.sh

```diff
@@ -80,6 +80,10 @@ pip3 install boto boto3 --user
 pip3 install 'yq==2.10.0' --user 
 
 
+# install AWS collection for ansible
+ansible-galaxy collection install amazon.aws
```

Comment:
@dvalinrh do we still need this or was it only for the days of yore when we were using the ansible cloud modules?

Reply:
No, we should not need that.  Recommendation is simply to comment it out for now.

Reply:
Turns out we still needed it.